cypress/support/step_definition/login_management/forcePasswordChangeSteps.spec.js

```diff
@@ -0,0 +1,25 @@
+import { defineStep } from "@badeball/cypress-cucumber-preprocessor";
```

Comment:
may this file also to admin folder?

Reply:
This happens during login, into the SVS, that's why I put it into the login management folder